packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -175,6 +175,32 @@ describe.each( [
 		expect( radio ).not.toBeChecked();
 	} );
 
+	if ( mode === 'controlled' ) {
```

Comment:
Should we move it next to the other `controlled`-only tests?

Reply:
No strong opinion, but I think I prefer it here because it's closely related to the test right above it.